src/main/java/com/textmessenger/service/EmailService.java

```diff
@@ -1,8 +1,14 @@
 package com.textmessenger.service;
 
+import com.textmessenger.config.AsyncConfiguration;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.textmessenger.config.AsyncConfiguration'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=242&issues=AWYNMafF_XXc3YMr1hF8&open=AWYNMafF_XXc3YMr1hF8)

[ISSUE_KEY]: AWYNMafF_XXc3YMr1hF8